Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -574,6 +576,20 @@ public ResultCode IsSixAxisSensorAtRest(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [CommandHipc(83)] // 6.0.0+
+        // IsFirmwareUpdateAvailableForSixAxisSensor(nn::hid::SixAxisSensorHandle, nn::hid::AppletResourceUserId, pid) -> bool UpdateAvailable
```

Comment:
Inverted parameters in definition
```suggestion
        // IsFirmwareUpdateAvailableForSixAxisSensor(nn::hid::AppletResourceUserId, nn::hid::SixAxisSensorHandle, pid) -> bool UpdateAvailable
```